conanfile.py

```diff
@@ -54,6 +54,9 @@ def _build_visual_studio(self):
             msbuild.build("lcms2.sln", targets=[target], platforms={"x86": "Win32"}, upgrade_project=upgrade_project)
 
     def _build_configure(self):
+        if self.settings.os == "Android" and tools.os_info.is_windows:
+            tools.replace_in_file(os.path.join(self._source_subfolder, 'configure'),
+                "s/[	 `~#$^&*(){}\\\\|;'\\\''\"<>?]/\\\\&/g", "s/[	 `~#$^&*(){}\\\\|;<>?]/\\\\&/g")
```

Comment:
what is this? consider adding some comment explaining what exactly changed here and why

Reply:
This is a pattern which add escape character '\\' to some special character, but when used on windows with ndk, we will get error like:

> lcms/2.9@test/test: libtool: compile:  /d/.conan/6fa54c/1/toolchains/llvm/prebuilt/windows-x86_64/bin/aarch64-linux-android26-clang.cmd -DPACKAGE_NAME=\\"lcms2\\" -DPACKAGE_TARNAME=\\"lcms2\\" -DPACKAGE_VERSION=\\"2.9\\" "-DPACKAGE_STRING=\\"lcms2 2.9\\"" -DPACKAGE_BUGREPORT=\\"\\" -DPACKAGE_URL=\\"\\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\\".libs/\\" -DHAVE_FUNC_ATTRIBUTE_VISIBILITY=1 -DHAVE_PTHREAD=1 -DHasTHREADS=1 -DHasZLIB=1 -I. -I../include -I../include -g --sysroot=/d/.conan/6fa54c/1/toolchains/llvm/prebuilt/windows-x86_64/sysroot -fPIC -fvisibility=hidden -c cmslut.c -o cmslut.o
> lcms/2.9@test/test: libtool: compile:  /d/.conan/6fa54c/1/toolchains/llvm/prebuilt/windows-x86_64/bin/aarch64-linux-android26-clang.cmd -DPACKAGE_NAME=\\"lcms2\\" -DPACKAGE_TARNAME=\\"lcms2\\" -DPACKAGE_VERSION=\\"2.9\\" "-DPACKAGE_STRING=\\"lcms2 2.9\\"" -DPACKAGE_BUGREPORT=\\"\\" -DPACKAGE_URL=\\"\\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\\".libs/\\" -DHAVE_FUNC_ATTRIBUTE_VISIBILITY=1 -DHAVE_PTHREAD=1 -DHasTHREADS=1 -DHasZLIB=1 -I. -I../include -I../include -g --sysroot=/d/.conan/6fa54c/1/toolchains/llvm/prebuilt/windows-x86_64/sysroot -fPIC -fvisibility=hidden -c cmsio1.c -o cmsio1.o
> lcms/2.9@test/test: =\"lcms2\""" was unexpected at this time.
> lcms/2.9@test/test: =\"lcms2\""" was unexpected at this time.                                                                          lcms/2.9@test/test: libtool: compile:  /d/.conan/6fa54c/1/toolchains/llvm/prebuilt/windows-x86_64/bin/aarch64-linux-android26-clang.cmd -DPACKAGE_NAME=\\"lcms2\\" -DPACKAGE_TARNAME=\\"lcms2\\" -DPACKAGE_VERSION=\\"2.9\\" "-DPACKAGE_STRING=\\"lcms2 2.9\\"" -DPACKAGE_BUGREPORT=\\"\\" -DPACKAGE_URL=\\"\\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\\".libs/\\" -DHAVE_FUNC_ATTRIBUTE_VISIBILITY=1 -DHAVE_PTHREAD=1 -DHasTHREADS=1 -DHasZLIB=1 -I. -I../include -I../include -g --sysroot=/d/.conan/6fa54c/1/toolchains/llvm/prebuilt/windows-x86_64/sysroot -fPIC -fvisibility=hidden -c cmsio0.c -o cmsio0.o
> lcms/2.9@test/test: =\"lcms2\""" was unexpected at this time.
> lcms/2.9@test/test: =\"lcms2\""" was unexpected at this time.
> lcms/2.9@test/test: =\"lcms2\""" was unexpected at this time.
> lcms/2.9@test/test: =\"lcms2\""" was unexpected at this time.                                                                         lcms/2.9@test/test: make[1]: *** [Makefile:503：cmscnvrt.lo] 错误 1
> lcms/2.9@test/test: make[1]: *** 正在等待未完成的任务....                                                               lcms/2.9@test/test: make[1]: *** [Makefile:503：cmserr.lo] 错误 1                                                       lcms/2.9@test/test: make[1]: *** [Makefile:503：cmsintrp.lo] 错误 1
> lcms/2.9@test/test: make[1]: *** [Makefile:503：cmsgmt.lo] 错误 1
> lcms/2.9@test/test: make[1]: *** [Makefile:503：cmsgamma.lo] 错误 1
> lcms/2.9@test/test: make[1]: *** [Makefile:503：cmsio1.lo] 错误 1
> lcms/2.9@test/test: make[1]: *** [Makefile:503：cmslut.lo] 错误 1                                                       lcms/2.9@test/test: make[1]: *** [Makefile:503：cmsio0.lo] 错误 1                                                       lcms/2.9@test/test: make[1]: 离开目录“/d/conan/.conan/data/lcms/2.9/test/test/build/742d5570dac6f0dca548ecabbe8e6261586f022e/source_subfolder/src”
> lcms/2.9@test/test: make: *** [Makefile:476：all-recursive] 错误 1
> lcms/2.9@test/test:
> lcms/2.9@test/test: ERROR: Package '742d5570dac6f0dca548ecabbe8e6261586f022e' build failed                              lcms/2.9@test/test: WARN: Build folder D:\\Conan\\.conan\\data\\lcms\\2.9\\test\\test\\build\\742d5570dac6f0dca548ecabbe8e6261586f022e
> ERROR: lcms/2.9@test/test: Error in build() method, line 72                                                                     self._build_configure()
> while calling '_build_configure', line 65
>         env_build.make()
>         ConanException: Error 2 while executing make -j8

The log above shows the compiler of ndk on windows may have a bug here which makes =\\"lcms2\\""" cause an error, so I remove the escape for character ' and " to make the compiler happy.